src/main.rs

```diff
@@ -16,12 +15,23 @@ async fn main() -> () {
         .await
         .unwrap_or_else(|_| config::incluster_config().expect("Failed to get local kube config and incluster config"));
 
-    let namespace = kube_config.default_ns.to_owned();
-    let client = APIClient::new(kube_config);
+    let controller = controller_init(kube_config, env_config).await.unwrap();
+
+    // Launch in new thread b/c HttpServer starts own async executor
+    let handle = std::thread::spawn(|| {
+        HttpServer::new(move || {
+            App::new()
+                .data(controller.clone())
+                .wrap(middleware::Logger::default().exclude("/health"))
+                .service(web::resource("/health").to(views::health))
+                .service(web::resource("/gordos").to(views::gordos))
+                .service(web::resource("/models").to(views::models))
+        })
+        .bind("0.0.0.0:8888")
```

Comment:
Maybe read this in from an env with this as default param?

Reply:
I agree!